charmcraft.yaml

```diff
@@ -9,3 +9,7 @@ bases:
     run-on:
     - name: ubuntu
       channel: "22.04"
+
+parts:
+  charm:
+    charm-binary-python-packages: [jsonschema]
```

Comment:
We need to install jsonschema (required by `auth_proxy` lib) before other requirements, otherwise packing the charm will fail.
I'm unable to find a related github issue but see [this](https://chat.charmhub.io/charmhub/pl/33pw9ryhufygupdnf4a9thh7ao) conversation for reference.